src/runner/mod.rs

```diff
@@ -96,6 +78,20 @@ impl Runner {
             }
         }
     }
+
+    fn prepare_before_run(&self) {
+        panic::set_hook(Box::new(|_panic_info| {
+            // XXX panics already catched at the test call site, don't output the trace in stdout
+        }));
+        let threads = if self.configuration.parallel { 0 } else { 1 }; // 0 is rayon default
+        let rayon_config = rayon::Configuration::new().num_threads(threads);
+        let _ = rayon::initialize(rayon_config);
```

Comment:
Frankly, I didn't touched it because I don't want to break anything, but I'd much rather don't call rayon at all when parallelism is disabled.